kcc-schema.json

```diff
@@ -0,0 +1,76 @@
+{
+   "$schema":"https://json-schema.org/draft/2020-12/schema",
+   "type":"object",
+   "properties":{
+      "credentialSubject":{
+         "type":"object",
+         "properties":{
+            "id":{
+               "type":"string"
+            },
+            "country_of_residence":{
```

Comment:
I would advise us to stick to a consistent casing mechanism
since the VCDM already uses snakeCase I would recommend updating this to countryOfResidence

Reply:
Do we need compliance to check off on if we can change from underscore to snakeCase?

Reply:
@nburgess48